nextmv/nextmv/cloud/application.py

```diff
@@ -871,6 +871,56 @@ def new_instance(
 
         return Instance.from_dict(response.json())
 
+    @staticmethod
```

Comment:
If it is a static method, shouldn’t we maybe have it as a convenience function outside of the class? Have a func like `def new_application(...) -> Application`? That way we don’t need the `"Application"` between quotes? After all, we are not using anything from the class, so maybe the method doesn’t belong there.

Reply:
Yeah, it really depends on where we want it to live. As a static on Application works for me, but an independently defined func is fine for me too. That would go on `cloud` package then, right?

So, either

```python
app = Application.new(...)
```

or 

```python
app = cloud.new_application(...)
```

Regarding the quotes around the type hint, that is still quite idiomatic Python, I think (based on [PEP 484](https://peps.python.org/pep-0484/#forward-references)). That doesn't mean that I lean towards it though. I don't have a favorite. :blush: 

Which one do you lean towards?

Reply:
Either works for me 🤷🏻‍♂️, that is why I approved. Whatever you decide, as you are the author!